arq/worker.py

```diff
@@ -295,13 +298,20 @@ def run(self) -> None:
             job_ids = await self.pool.zrangebyscore(
                 self.queue_name, offset=self._queue_read_offset, count=count, max=now
             )
+            self.abort_tasks = [
+                k[len(abort_key_prefix):]
+                for k in await self.pool.keys('%s*' % abort_key_prefix)
+            ]
         await self.run_jobs(job_ids)
 
-        for t in self.tasks:
-            if t.done():
-                self.tasks.remove(t)
+        jobs = copy(self.tasks)
```

Comment:
this can be inlined, probably as `list(jobs.items())`

Reply:
ok